variables.tf

```diff
@@ -148,3 +148,9 @@ variable "disable_dependent_services" {
   type        = string
 }
 
+variable "shared_vpc_enabled" {
```

Comment:
This variable should not be defined here.

Reply:
Removed